chalicelib_fourfront/check_setup.json

```diff
@@ -1291,6 +1291,21 @@
             }
         }
     },
+    "hichip_status": {
```

Comment:
I do not understand why this diff is showing. This file is the same as what's in the master branch.